### DISCUSSION TITLE
Cannot change resolution on Wayland

### DISCUSSION THREAD

**Discussion by 25pwn** [ppy/osu#16766] [Ideas] [answered]
Changing resolutions on X11 works fine. On Wayland, it just graphically glitches for a second and has no other effect.
I'm running a KDE Wayland session on Arch Linux.

**Comment by peppy** [comment, marked as answer]
might be best reporting to sdl. we can't really deal with per installation linux issues.

**Reply by 25pwn** [reply]
I found a solution, SDL_VIDEODRIVER=wayland fixes this.
